### ISSUE TITLE
Downloaded file appended for duplicate instances of script

### ISSUE THREAD

**Issue by vNa5h** [Akianonymus/gdrive-downloader#23]
First of all, this is a great tool, helps us download folders in quite easier way.

While downloading the internet connection was disconnected. Which made the script skip some files.
Like file1 was 1gb only 200mb was downloaded.
And file2 downloading had begun.
So I launched the script again.
Now file 1 and 2 was complete.
But file 3 which was 1gb was downloaded twice. The completed size was 2gb

So in short

> There should be size check or file integrity check.

> A way to kill the script as killing curl makes it launch again automatically somehow

**Comment by Akianonymus** [comment]
Update to the latest version by gdl -u and check if this issue persists.
